CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.10)
 project(MunRuntime VERSION 0.1.0 LANGUAGES C CXX)
 
+option(mun_build_examples "Build all of Mun's own examples." OFF)
```

Comment:
Why the change of capitalization? I always use capital variable names. I think thats common practice but cant find the source.

Reply:
I saw that we used a mix of capitalised and non-capitalised and while looking at `googletest` noticed that they use lowercase too, so went with that.

For reference, https://github.com/google/googletest/blob/master/googletest/CMakeLists.txt

Reply:
Test webhook